docs/pages/_app.tsx

```diff
@@ -1,18 +1,19 @@
 import { Header } from '../components/Header/Header';
+import { Footer } from '../components/Footer/Footer';
 import type { AppProps } from 'next/app';
 import * as React from 'react';
 import '../styles/globals.css';
 
 const App: React.FC<AppProps> = ({ Component, pageProps }) => {
-    return (
-        <>
-          <Header />
-          <main className='container'>
-            <Component {...pageProps} />
-          </main>
-        </>
-    )
-    
+  return (
+      <div className="flex h-screen w-screen flex-col justify-between">
```

Comment:
I don't think Prettier is running automatically because this wouldn't be indented so much.